PublishParasoftResults/ParaReportPublishService.ts

```diff
@@ -891,14 +885,10 @@ export class ParaReportPublishService {
         return uuid.v5(violType + ruleId + msg + severity + lineHash + uri + order, namespace);
     }
 
-    private isStaticAnalysisReportNameDuplicate = (sourcePath: string): boolean => {
-        const filename = path.basename(sourcePath);
-        if (this.staticAnalysisReportsMap.has(filename)) {
-            tl.warning(`Skipping report with duplicate names: ${sourcePath}`);
-            return true;
-        }
-        this.staticAnalysisReportsMap.set(filename, sourcePath);
-        return false;
+    private renameSarifReport = (sourcePath: string): string => {
```

Comment:
change the name to makeFileNameUnique, and this function can be moved to a new ts file(name is StaticAnalysisReportService).